src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,95 @@
+package edu.kis.vh.nursery;
+
+/**
+ * Klasa DefaultCountingOutRhymer reprezentuje prosty mechanizm liczenia.
+ * Ta klasa umożliwia dodawanie liczb do wewnętrznej pamięci (stos),
+ * sprawdzanie, czy jest pełna, sprawdzanie, czy zawiera jakiekolwiek liczby oraz usuwanie (liczenie) liczb.
+ */
+public class DefaultCountingOutRhymer {
+
+    /**
+     * Maksymalna liczba elementów, które rhymer może przechować.
+     */
+    private static final int SIZE = 12;
+
+    /**
+     * Tablica, która przechowuje liczby w rhymerze.
+     */
+    private int[] numbers = new int[SIZE];
+
+    /**
+     * Kod błędu zwracany przez metody, gdy operacja jest niewłaściwa.
+     */
+    private static final int ERR_CODE = -1;
+
+    /**
+     * Indeks reprezentujący aktualną pozycję w rhymerze.
+     * Inicjowany wartością ERR_CODE, aby wskazywać, że rhymer jest pusty.
+     */
+    private int total = ERR_CODE;
+
+    /**
+     * Zwraca wartość indeksu total, który reprezentuje aktualną pozycję w rhymerze.
+     * Używane do określenia, ile liczb jest przechowywanych.
+     *
+     * @return aktualna pozycja (total).
+     */
+    public int getTotal(){
+        return total;
+    }
+
+    /**
+     * Dodaje liczbę do rhymera, jeśli nie jest pełny.
+     * Jeśli rhymer jest pełny, ta metoda nic nie robi.
+     *
+     * @param in liczba do dodania do rhymera.
+     */
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    /**
+     * Sprawdza, czy rhymer jest pusty.
+     * Rhymer jest uznawany za pusty, jeśli total ma wartość ERR_CODE.
+     *
+     * @return true, jeśli rhymer jest pusty, w przeciwnym razie false.
+     */
+    public boolean callCheck() {
+        return total == ERR_CODE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.